docs/xgcpolicy.md

```diff
@@ -106,6 +106,9 @@ The behavior of the following options is different when specified with `-Xgcpoli
 [`-Xcompactexplicitgc`](xcompactexplicitgc.md) (default)
 : Forces compaction in explicit Global GC cycles, such as those invoked by `System.gc()`. Compaction in implicit Global GC remains optional, triggered by internal heuristics.
 
+[`-Xgc:targetPausetime`](xgc.md#targetpausetime) 
```

Comment:
Putting this between `-Xcompactexplicitgc` and `-Xnocompactexplicitgc` isn't appropriate, the options aren't listed alphabetically.